desync.c

```diff
@@ -143,22 +141,19 @@ void wait_send(int sfd)
 #define wait_send_if_support(sfd) // :(
 #endif
 
-#ifdef FAKE_SUPPORT
```

Comment:
Is this based on older version of file or reverts changes?

Reply:
> reverts changes

The function has a lot of Linux-specific calls, so I decided to put everything in one ifdef